src/utils.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import print_function
+
+import os
+import io
+import sys
+import imp
+import json
+from os.path import join, exists
+from collections import OrderedDict
+
+ENCODING = 'ISO-8859-1'
+
+
+def is_empty(val):
+    """Check if a variable is an empty string or an empty list."""
+    return val == "" or val == []
```

Comment:
We can make this Idiomatic
 def is_empty(val):
    """Check if a variable is an empty string or an empty list."""
    return not val